Mathlib/AlgebraicGeometry/Spec.lean

```diff
@@ -228,63 +267,84 @@ def Spec.locallyRingedSpaceMap {R S : CommRingCat} (f : R ⟶ S) :
   LocallyRingedSpace.Hom.mk (Spec.sheafedSpaceMap f) fun p =>
     IsLocalRingHom.mk fun a ha => by
       -- Here, we are showing that the map on prime spectra induced by `f` is really a morphism of
-      -- *locally* ringed spaces, i.e. that the induced map on the stalks is a local ring homomorphism.
-      rw [← local_ring_hom_comp_stalk_iso_apply] at ha 
-      replace ha := (stalk_iso S p).Hom.isUnit_map ha
-      rw [iso.inv_hom_id_apply] at ha 
-      replace ha := IsLocalRingHom.map_nonunit _ ha
-      convert RingHom.isUnit_map (stalk_iso R (PrimeSpectrum.comap f p)).inv ha
-      rw [iso.hom_inv_id_apply]
+      -- *locally* ringed spaces, i.e. that the induced map on the stalks is a local ring
+      -- homomorphism.
+      erw [← localRingHom_comp_stalkIso_apply] at ha
+      replace ha := (stalkIso S p).hom.isUnit_map ha
+      rw [← comp_apply, show localizationToStalk S p = (stalkIso S p).inv from rfl,
+        Iso.inv_hom_id, id_apply] at ha
+      replace ha := IsLocalRingHom.map_nonunit
```

Comment:
```suggestion
      -- Porting note: `R` and `S` had to be made explicit
      replace ha := IsLocalRingHom.map_nonunit
```

Reply:
After second check, only `R` needs to be made explicit